client/src/index.js

```diff
@@ -1,7 +1,8 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import { BrowserRouter as Router } from 'react-router-dom';
 
 import './index.css';
 import App from './App';
 
-ReactDOM.render(<App />, document.getElementById('root'));
+ReactDOM.render(<Router><App /></Router>, document.getElementById('root'));
```

Comment:
```suggestion
ReactDOM.render(
    <Router>
        <App />
    </Router>, 
    document.getElementById('root'));
```
again, break the code into multiple lines